testing/pluto/TESTLIST

```diff
@@ -724,6 +724,8 @@ kvmplutotest	ikev2-child-rekey-09-windows	wip	github:576
 kvmplutotest	ikev2-child-rekey-10-impair-rekey-initiate-subnet	good
 kvmplutotest	ikev2-child-rekey-10-impair-rekey-respond-supernet	good
 kvmplutotest	ikev2-child-rekey-10-impair-rekey-respond-subnet	good
+kvmplutotest	ikev2-expire-01-bytes		good
+kvmplutotest	ikev2-expire-01-packets		good
```

Comment:
would be good to add a test case with ipcomp because that is also weird and might cause its own expire event too?